### ISSUE TITLE
app.put() body is not parsed

### ISSUE THREAD

**Issue by Nosthertus** [adamhalasz/diet#54]
running on `Diet 0.10.8`, just making the First PUT i have ever done on a server and i have found out that `$.body` on this method is not being parsed, the whole data is still serialized

**Comment by adamhalasz** [comment]
Hey @Nosthertus is it a multipart request?

**Comment by Nosthertus** [comment]
@adamhalasz no, it's just a simple PUT request, with no upload file, just like a normal POST
